@xen-orchestra/fs/src/index.js

```diff
@@ -1,24 +1,36 @@
 // @flow
+import execa from 'execa'
 
 import type RemoteHandler from './abstract'
 import RemoteHandlerLocal from './local'
 import RemoteHandlerNfs from './nfs'
 import RemoteHandlerSmb from './smb'
+import RemoteHandlerSmbMount from './smb-mount'
 
 export type { default as RemoteHandler } from './abstract'
 export type Remote = { url: string }
 
 const HANDLERS = {
   file: RemoteHandlerLocal,
-  smb: RemoteHandlerSmb,
   nfs: RemoteHandlerNfs,
 }
 
 export const getHandler = (remote: Remote, ...rest: any): RemoteHandler => {
   // FIXME: should be done in xo-remote-parser.
   const type = remote.url.split('://')[0]
 
-  const Handler = HANDLERS[type]
+  let Handler
+  if (type === 'smb') {
+    try {
+      execa.sync('mount.cifs', ['-V'])
```

Comment:
Move it out of `getHandler`, let's do it once and for all when the module loads.